From: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Subject: tools: fix double header include
Date: Thu, 25 Jul 2024 10:47:40 +0100

We already imported the stubbed-out linux/types.h. This has no impact as it
has a header guards but let's clean this up.

Link: https://lkml.kernel.org/r/7aae7eb2-6170-4ed6-bee5-6a0c35294381@lucifer.local
Signed-off-by: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/shared/shared.h |    1 -
 1 file changed, 1 deletion(-)

--- a/tools/testing/shared/shared.h~tools-separate-out-shared-radix-tree-components-fix
+++ a/tools/testing/shared/shared.h
@@ -6,7 +6,6 @@
 #include <linux/bitops.h>
 
 #include <linux/gfp.h>
-#include <linux/types.h>
 #include <linux/rcupdate.h>
 
 #ifndef module_init
_
